feature/home/src/main/java/com/epmedu/animeal/home/data/model/Feeder.kt

```diff
@@ -0,0 +1,7 @@
+package com.epmedu.animeal.home.data.model
+
+data class Feeder(
```

Comment:
Why data class should have values by default?

Reply:
Updated